Dockerfile

```diff
@@ -1,12 +1,10 @@
-FROM node:boron
+FROM node:boron-slim
 LABEL maintainer Mofesola Babalola <me@mofesola.com>
 
 #Get required applications
 ENV DEBIAN_FRONTEND noninteractive
-RUN apt-get update && apt-get install -y  curl
 
```

Comment:
You may want to leave the curl installation here as it is being actively used in the entire setup. See [here](https://github.com/HospitalRun/hospitalrun-server/blob/master/conf/initcouch.sh)